packages/react-button/src/components/CommandBarButton/CommandBarButton.scss

```diff
@@ -0,0 +1,48 @@
+@import '~@fluentui/common-styles/dist/sass/themeCssVariables';
+@import '~@fluentui/common-styles/dist/sass/highContrast';
+@import '~@fluentui/common-styles/dist/sass/focusBorder';
+
+.root {
+  --button-background: var(--palette-white);
+  --button-contentColor: var(--palette-neutralPrimary);
+  --button-borderWidth: 0;
+  --button-borderRadius: 0;
+
+  @include fonts-medium;
+
+  @include focus-border($padding: 3, $color: transparent);
+
+  @include high-contrast {
+    @include focus-border($padding: 4, $thickness: 0, $color: transparent);
+    $border: none;
+  }
+
+  &._pressed {
+    --button-background: var(--palette-neutralLight);
```

Comment:
you shouldn't be using any `--palette-` tokens. v8 tokens should be defined in `createDefaultTheme.ts`